src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,69 @@
+package study;
+
+import static org.assertj.core.api.Assertions.*;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class SetTest {
+	private Set<Integer> numbers;
+
+	@BeforeEach
+	void setUp() {
+		numbers = new HashSet<>();
+		numbers.add(1);
+		numbers.add(1);
+		numbers.add(2);
+		numbers.add(3);
+	}
+
+	@Test
+	@DisplayName("size()를 통해 Set의 크기를 확인할 수 있다")
+	void size() {
+		// Act
+		int size = numbers.size();
+
+		// Assert
+		assertThat(size).isEqualTo(3);
+	}
+
+	@ParameterizedTest
+	@ValueSource(ints = {1, 2, 3})
+	@DisplayName("contains()를 통해 1, 2, 3의 값이 존재하는지를 확인할 수 있다")
+	void contains(
+		int value
+	) {
+		// Act
+		boolean contains = numbers.contains(value);
+
+		// Assert
+		assertThat(contains).isTrue();
+	}
```

Comment:
```suggestion
	void contains(int value) {
        assertThat(numbers).contains(value);
    }
```

test case가 단순한 경우 굳이 라인수를 늘리기 보다 가능한 적은 라인으로 구현하는 것은 어떨까?